Rakefile

```diff
@@ -12,9 +11,8 @@ end
 task default: ['spec']
 
 spec = Gem::Specification.new do |s|
-
   s.name              = 'registers-ruby-client'
-  s.version           = RegistersClient::VERSION
+  s.version           = File.read('.ruby-version').chomp
```

Comment:
As mentioned above, this can't be consuming `.ruby-version`. Typically you find a structure such as 

```
lib/
+- registers-client
     +- version.rb
     +- ...
```

With the content for `version.rb`:

```ruby
module RegistersClient
  VERSION = "0.2.0"
end
```

Reply:
Why can't this be consuming `.ruby-version`? Perhaps there is some confusion, in that `.ruby-version` isn't the version of Ruby being used but is the versioning of the library. I will rename this file to make this point clear.